include/functions_entries_admin.inc.php

```diff
@@ -104,10 +104,14 @@ function serendipity_printEntryForm($targetURL, $hiddens = array(), $entry = arr
 
             $template_vars['category_options'][] = $cat;
         }
+    } else {
+        $template_vars['category_options'] = [];
     }
 
     if (!empty($serendipity['GET']['title'])) {
```

Comment:
As I read this, this looks a bit XSS-prone to me, might need to check this?!

Reply:
Could you explain a bit more - you probably do no mean this assignment, right?

Reply:
I meant the `url?serendipity['GET']['title']=<strong>evilHTML</strong>` possibility maybe, because the lines after this seem to pass through a GET variant (don't know if we really read from GET or if we just abused it as a superglobal to store a database title in)

Reply:
Testing this, the title seems to be properly escaped when set into the form later on.